deploy/terraform/variables.tf

```diff
@@ -2,327 +2,368 @@
-}
-variable "GP2GP_GPC_OVERRIDE_FROM_ASID" {
-        type=string
-}
-variable "GP2GP_GPC_OVERRIDE_NHS_NUMBER" {
-        type=string
-}
-variable "GP2GP_GPC_OVERRIDE_TO_ASID" {
-        type=string
+  type = string
 }
+
 variable "GP2GP_GPC_STRUCTURED_FHIR_BASE" {
-        type=string
+  type = string
 }
+
 variable "GP2GP_LARGE_ATTACHMENT_THRESHOLD" {
-        type=string
+  type = string
 }
+
 variable "GP2GP_LARGE_EHR_EXTRACT_THRESHOLD" {
-        type=string
+  type = string
 }
+
 variable "GP2GP_LOGGING_LEVEL" {
-        type=string
+  type    = string
+  default = "DEBUG"
 }
 variable "GP2GP_MHS_INBOUND_QUEUE" {
-        type=string
+  type = string
 }
 variable "GP2GP_MHS_OUTBOUND_URL" {
-        type=string
+  type = string
 }
 variable "GP2GP_MONGO_DATABASE_NAME" {
-        type=string
+  type = string
 }
 variable "GP2GP_MONGO_TTL" {
-        type=string
+  type = string
 }
 variable "GP2GP_MONGO_URI" {
-        type=string
+  type = string
 }
 variable "GP2GP_SERVER_PORT" {
-        type=string
-}
-variable "GP2GP_SSL_TRUST_STORE_URL" {
-        type=string
+  type    = number
+  default = 8183
 }
+
 variable "GP2GP_STORAGE_CONTAINER_NAME" {
-        type=string
-        default = null
+  type    = string
+  default = "gp2gp-storage-bucket"
 }
+
 variable "GP2GP_STORAGE_TYPE" {
-        type=string
-        default = null
+  type    = string
+  default = null
 }
+
 variable "GP2GP_TRANSLATOR_SERVER_PORT" {
-        type=string
-}
-variable "GPC_CONSUMER_GPC_GET_DOCUMENT_PATH" {
-        type=string
-}
-variable "GPC_CONSUMER_GPC_GET_PATIENT_PATH" {
-        type=string
-}
-variable "GPC_CONSUMER_GPC_STRUCTURED_PATH" {
-        type=string
+  type = string
 }
+
 variable "GPC_CONSUMER_LOGGING_LEVEL" {
-        type=string
+  type    = string
+  default = "DEBUG"
 }
+
 variable "GPC_CONSUMER_SDS_APIKEY" {
-        type=string
+  type    = string
+  default = "anykey"
 }
+
 variable "GPC_CONSUMER_SDS_URL" {
-        type=string
-}
-variable "GPC_CONSUMER_SEARCH_DOCUMENTS_PATH" {
-        type=string
+  type    = string
+  default = "http://localhost:8080/spine-directory/"
 }
+
 variable "GPC_CONSUMER_SERVER_PORT" {
-        type=string
+  type    = number
+  default = 8090
 }
+
 variable "GPC_CONSUMER_SPINE_CLIENT_CERT" {
-        type=string
-        default = null
-}
-variable "GPC_CONSUMER_SPINE_CLIENT_KEY" {
-        type=string
-        default = null
-}
-variable "GPC_CONSUMER_SPINE_ROOT_CA_CERT" {
-        type=string
-        default = null
-}
-variable "GPC_CONSUMER_SPINE_SUB_CA_CERT" {
-        type=string  
-        default = null
-}
-variable "GPC_CONSUMER_SSP_URL" {
-        type=string
-        default = null
-}
-variable "GPC_FACADE_SERVER_PORT" {
-        type=string
+  type    = string
+  default = null
 }
+
 variable "MAX_RESYNC_RETRIES" {
-        type=string
+  type    = number
+  default = 3
 }
+
 variable "MHS_AMQP_BROKER" {
-        type=string
+  type    = string
+  default = "amqp://activemq:5672"
```

Comment:
This value looks a little suspicious to me.  How does the DNS resolve for `activemq` hostname ?

Reply:
I took this name from Danny's vars.tfvars file and thought that it would be a good standard default value and a basis for adjustments if anyone wants to replace it with a custom name.

Reply:
We are actually not using this Variable anymore from what I can see:
![image](https://github.com/NHSDigital/patient-switching-standard-adaptors-architecture/assets/101815369/5801f9c9-bd0e-43a6-8bf3-d32b3f40b19b)
As the above image shows we are instead using the AWS MQ Broker, when we started development on this we initially had an ActiveMQ Image we were using within Fargate but it was causing too many issues hence we decided to switch to the AWS solution.

Reply:
ok, I will delete this default value then.

Reply:
I think @dfk97 point was that the variable could be removed in its entirety, not just the default value.

Reply:
I think it is used in two places in main.tf file hence we can't delete them unless we can delete it all together with the variables that are defined in main.tf if they are not needed. @dfk97 could you confirm if MHS_AMQP_BROKER and MHS_INBOUND_QUEUE_BROKERS variables are still needed in main.tf ?